src/detail-page/DetailPage.js

```diff
@@ -59,37 +59,44 @@ function DetailSection({
   title,
   description,
   showDimensionControl,
+  showDistrictControl,
   showTimeControl,
 }) {
-  const showControls = showDimensionControl || showTimeControl;
-
   const [dimension, setDimension] = useState();
   const [month, setMonth] = useState();
 
   // these are placeholders; replace them with values derived from data
   const [endDate] = useState(new Date());
   const [startDate] = useState(subYears(endDate, 3));
 
+  // this is also a placeholder;
+  // once data is loaded this should be updated with a list of districts
```

Comment:
I should note that even on Pulse Dashboard, the set of districts is currently hard-coded to avoid any really odd UX happening from temporary blips in the data, or e.g. from a new district showing up without us having had a chance to vet that it _should_ be included in visualizations. I'd suggest we do the same thing here.

Reply:
yeah i am seeing that now that i'm looking at the data more closely .... laying tracks right in front of the train here. I will plan to follow pulse dashboard on that